database/sql/src/main/scala/cromwell/database/slick/WorkflowStoreSlickDatabase.scala

```diff
@@ -88,13 +88,25 @@ trait WorkflowStoreSlickDatabase extends WorkflowStoreSqlDatabase {
       } yield ()
     }
 
+    def fetchAndUpdateStartableWfs(hogGroup: Option[String]) = {
+      for {
+        workflowStoreEntries <- dataAccess.fetchStartableWfsForHogGroup(
+          (limit.toLong, heartbeatTimestampTimedOut, workflowStateExcluded, hogGroup)
+        ).result
+        _ <- DBIO.sequence(
+          workflowStoreEntries map updateForFetched(cromwellId, heartbeatTimestampTo, workflowStateFrom, workflowStateTo)
```

Comment:
The `updateForFetched` code is same as before. So based on [this code](https://github.com/broadinstitute/cromwell/blob/24dff0e71e9b2b29cfff45e3d8b9baeff44fd4e0/database/sql/src/main/scala/cromwell/database/slick/WorkflowStoreSlickDatabase.scala#L73-L76), it seems that if the workflow was in `Submitted` state it would be updated to `Running` state, otherwise it will have same state as it was previously in.